BuildDetails.md

```diff
@@ -0,0 +1,77 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how projects are built and why they are built that way. Please note that this guide only fully applies to the GameStartr games, as some of these tasks are not run when building the modules.
+
+### Table of Contents
+
+1. [How to Build](#how-to-build)
+2. [Grunt Tasks](#grunt-tasks)
+
+## How to Build
+In order to build a FullScreenShenanigans project you must use Grunt. To do so, open the terminal and navigate to the project directory. You can build the project all at once or step-by-step.
+
+If you wish to build it in one go, enter `grunt`. This command will run all of the tasks specified in the [Grunt Tasks](#grunt-tasks) section of this document.
+
+Alternatively, you can run each grunt task manually. This allows you to run part of the build at a time. In order to run an individual task, type `grunt task_name` and press enter. For example, if you want to run the clean task, enter `grunt clean`. The different tasks are outlined in the [Grunt Tasks](#grunt-tasks) section of this document.
+
+
+## Grunt Tasks
+The following ten tasks are run, in order, every time you build the project with grunt. The end result of the build is a new directory named Distribution. This directory will contain all of the project files in minified form.
+
+### clean
+The clean task deletes all files and folders in the project's Distribution directory. This occurs because the Distribution folder must be recreated to reflect any changes made to the files.
+
+See the [grunt-contrib-clean page](https://github.com/gruntjs/grunt-contrib-clean) for more information on the task.
```

Comment:
For this and the later ones, you can remove "on the task" / "on X" from the end.

When it's intentionally the same short blurb in every one it's better to be exactly the same. Variety is good for larger paragraphs.